src/types/onyx/ReportNextStep.ts

```diff
@@ -0,0 +1,50 @@
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+
+type Message = {
+    text: string;
+    type: string;
+    action: string;
+};
```

Comment:
```suggestion
type Message = {
    text: string;
    type?: string;
    action?: string;
};
```

This type is already defined [here](https://github.com/Expensify/App/blob/main/src/libs/NextStepUtils.ts), but we should leave in this file. Could you apply the changes above and make `NextStepUtils` file use this one?